client/knowledge_base_search/src/containers/SearchContainer/styles.css

```diff
@@ -49,17 +127,24 @@ button.random-query--button {
 }
 
 
-button.random-query--button:hover,
-button.random-query--button:focus {
+button.random-query--button:hover {
   color: #003C32;
   border-color: #003C32;
 }
 
+button:focus {
```

Comment:
Applying this to all buttons provides consistency across the demo. Also, these are the only buttons to date in the demo: there are only links and text searches otherwise.